evacuees/src/UI/embc-evacuee/src/app/sharedModules/components/evacuation-file/evacuation-file-list/evacuation-file-list.component.ts

```diff
@@ -35,7 +36,7 @@ export class EvacuationFileListComponent implements OnInit {
       this.showLoading = true;
       this.evacuationFileService.getCurrentEvacuationFiles().subscribe(files => {
         this.dataSourceActive = files;
-        this.dataSourceActive.sort((a, b) => parseInt(b.essFileNumber, 10) - parseInt(a.essFileNumber, 10));
+        this.dataSourceActive.sort((a, b) => new Date(b.evacuationFileDate).valueOf() - new Date(a.evacuationFileDate).valueOf());
```

Comment:
sorting/pagination logic should preferably be sitting in ngAfterViewInit() to make sure it works all the time. Is the sorting for all table headers or just for Date?